docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+version: "3"
```

Comment:
I keep getting this error when I try to run docker-compose up which might be something wrong with my computer and docker! or an indentation error apparently? 

expected <block end>, but found '<block mapping start>'
  in "./docker-compose.yaml", line 15, column 3

Reply:
I think maybe "client" needs to be indented two more spaces?